tasks/aem-author-tests.yml

```diff
@@ -14,4 +14,4 @@
     port: "{{ conga_config.quickstart.port }}"
     flush_target: "{{ conga_config.replication.author.flushTarget }}"
     type: "author"
-  when: conga_config.replication.author.flushTarget
+  when: conga_config.replication.author.flushTarget.name
```

Comment:
Maybe we should test for `conga_config.replication.author.flushTarget`  and `conga_config.replication.author.flushTarget.name` here?

Reply:
Why? The included tasks are completely dependent on `conga_config.replication.author.flushTarget.name`, nothing else is being used in there. If this one is defined, it's clear that `conga_config.replication.author.flushTarget` exists, too. Besides that, nothing but `conga_config.replication.author.flushTarget.name` is used in the included tasks

Reply:
Ah, I just saw why: line 15. ;) I will add this.